sr_object_symmetry/launch/sr_object_symmetry.launch

```diff
@@ -0,0 +1,60 @@
+<launch>
+  <!-- Rotational parameters -->
+  <arg name="ref_max_fit_angle" default="45.0"/> <!-- Refinement of initial symmetries -->
+  <arg name="min_normal_fit_angle" default="10.0"/> <!-- minimum fit angle between a symmetry and a point -->
+  <arg name="max_normal_fit_angle" default="60.0"/> <!-- maximum fit angle between a symmetry and a point -->
+  <arg name="max_symmetry_score" default="0.02"/> <!-- Maximum score to be accepted (lower has better match) -->
+  <arg name="max_perpendicular_score" default="0.6"/> <!-- perpendicularity scores for individual points -->
+  <arg name="min_coverage_score" default="0.3"/> <!-- The minimum score for coverage for filtering symmetries -->
+  <!-- Reflectional parameters -->
+  <arg name="voxel_size" default="0.0"/> <!-- Downsample -->
+  <arg name="num_angle_divisions" default="20"/> <!-- number of divisions of a sphere -->
+  <arg name="flatness_threshold" default="0.005"/> <!-- maximum third eigenvalue for the segment to be considered flat -->
+  <arg name="refine_iterations" default="20"/> <!-- maximum number of optimization iterations -->
+  <arg name="max_correspondence_reflected_distance" default="0.01"/> <!-- maximum distance between the first point of a symmetric
+  correspondence and a reflection of the second point -->
+  <arg name="min_inlier_normal_angle" default="15.0"/> <!-- minimum normal fit angle for an inlier symmetric correspondence -->
+  <arg name="max_inlier_normal_angle" default="20.0"/> <!-- maximum normal fit angle for an inlier symmetric correspondence -->
+  <arg name="min_cloud_inlier_score" default="0.15"/> <!-- Symmetry filtering parameters -->
+  <arg name="min_corresp_inlier_score" default="0.9"/> <!-- Symmetry filtering parameters -->
+  <arg name="symmetry_min_angle_diff" default="7.0"/> <!-- Two symmetries are considered similar if the angle between their normals
+  is less than this threshold -->
+  <arg name="symmetry_min_distance_diff" default="0.01"/> <!-- Two symmetries are considered similar if the difference between
+  their distances to origin is less than this threshold -->
+  <arg name="max_reference_point_distance" default="0.3"/> <!-- Maximum distance between the reference points of two symmetries
+  that can be merged -->
+
+  <!-- General parameters -->
+  <arg name="visualization" default="true"/> <!-- This will show you the symmetries and prompt you to accept/reject -->
+  <arg name="overwrite" default="true"/> <!-- This will overwrite the current yaml files -->
```

Comment:
Please explain behaviour when `false`: Generation will be skipped for existing yaml files and carried out normally for non-existing ones.

Reply:
done